src/agentscope/models/dashscope_model.py

```diff
@@ -197,6 +208,32 @@ def __call__(
             raw=response,
         )
 
+    def format(
+        self,
+        *args: Union[Msg, Sequence[Msg]],
+    ) -> List:
+        """Format the messages for DashScope Chat API.
+
+        Args:
+            *args (`Union[Msg, Sequence[Msg]]`):
+                The input arguments to be formatted, where each argument
+                should be a `Msg` object, or a list of `Msg` objects
+
+        Returns:
+            `List[dict]`:
+                The formatted messages.
+        """
+        # TODO: This function only convert agentscope msgs into qwen
+        #  messages, the re-range is executed in _preprocess_role function.
```

Comment:
Since the returned list will always have length 1, maybe here we need to make it more clear.

Reply:
Add more detailed docstring